### PR TITLE
Minor improvements to the logging UI

### PR DESCRIPTION
**Pull request by daniel-beck**
## Before

> Screenshot 2019-10-19 at 20 44 38

## After

> Screenshot 2019-10-19 at 20 44 22

> Screenshot 2019-10-19 at 20 44 13

The last screenshot documents https://github.com/jenkinsci/jenkins/commit/010d9e9067b7fa3ce776ec5f76a6cc4d0fdff258

### Proposed changelog entries

* Minor UI improvements in the System Log.

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### CODE REVIEW

**Review by res0nance — APPROVED** [review]

File: core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly]:
```suggestion
      Log messages at a level lower than INFO are never recorded in this default Jenkins log. Use <a href="/">log recorders</a> to record these log messages.
```

**Review by oleg-nenashev — APPROVED** [review]
I like the text improvements, so approving it. The side panel was better before the change imho

**Comment by daniel-beck** [comment]
> The side panel was better before the change imho

What specifically? Labels (which?), order, the "Add" entry being a sub-item?

**Comment by oleg-nenashev** [comment]
Yes, I was concerned about the sub-irem's loo and feel only. Even in my slidedecks I try to avoid the second tabulation level now

**Review by fcojfernandez — APPROVED** [review]

**Review by alecharp — APPROVED** [review]
Great improvement indeed. Thanks @daniel-beck !

File: core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly
Comment (jsoref) [reply core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly]: I think the link shouldn't go to `/`

Comment (daniel-beck) [reply core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly]: Good point, this seems wrong. I think I wanted it to go to `.` due to how weird the entire logging UI is structured, which would be a straightforward typo.

File: core/src/main/resources/hudson/logging/Messages.properties
Comment (jsoref) [inline core/src/main/resources/hudson/logging/Messages.properties]: Can you add a new line at eof to make tooling happy?

Comment (oleg-nenashev) [reply core/src/main/resources/hudson/logging/Messages.properties]:
```suggestion
LogRecorderManager.DisplayName=System Log

```

File: core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly
Comment (oleg-nenashev) [reply core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly]:
```suggestion
      Log messages at a level lower than INFO are never recorded in the Jenkins system log. Use <a href=".">log recorders</a> to record these log messages.
```

**Comment by oleg-nenashev** [comment]
I think comments from @jsoref worth addressing before the merge

File: core/src/main/resources/hudson/logging/Messages.properties
Comment (daniel-beck) [reply core/src/main/resources/hudson/logging/Messages.properties]: My tooling is quite happy as is. As is Jenkins', this file has never had a trailing newline.

File: core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly
Comment (daniel-beck) [reply core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly]: It's a bit more complicated than that.

The breadcrumb you're referring to now matches the label on `/manage`, which was the goal of the change.

The `LogRecorderManager` UI includes both what is called "Jenkins log" here, and "All Log Messages" (previously "All Logs") in the sidepanel, as well as log recorders and the global levels overview.

So if anything should be changed, it's that this doesn't say "Log messages … in all log messages" which I didn't want to write, because it makes no sense at all.

In the interest of limiting the scope of this PR, I wrote a message I hoped could be understood, and moved on.

File: core/src/main/resources/hudson/logging/Messages.properties
Comment (jsoref) [reply core/src/main/resources/hudson/logging/Messages.properties]: Git/GitHub don't really like files w/o them.

**Comment by oleg-nenashev** [comment]
@jsoref Do you agree with responses from @daniel-beck ?

**Comment by jsoref** [comment]
I'm happy w/ incremental improvements. I don't know that I understand this code enough to be able to make them, so I'm happy to defer to @daniel-beck.

I think we've both identified that the UI has inconsistent overlapping elements which probably need to be detangled at some point, but it doesn't have to be done today and again would require someone who really understands this stuff to work on it.

**Comment by oleg-nenashev** [comment]
Thanks @jsoref ! I will go ahead and merge it then

**Pull request merged**
